test/test_manifest.json

```diff
@@ -6583,5 +6583,34 @@
       "rounds": 1,
       "type": "eq",
       "annotations": true
+   },
+   { "id": "bug1675139",
+      "file": "pdfs/bug1675139.pdf",
```

Comment:
Also, this file doesn't seem to be included in the patch :-)

Reply:
pffff....